README.md

```diff
@@ -1 +1,205 @@
+> Describe between 1-3 key features of the application.
+
+- it's a basic web page using HTML and CSS
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://artufelii.github.io/microverse-hello-world/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+- download the git repository 
+- run npm init -y to create a node_modules folder
+- add package.json to your root directory 
+- run npm install for install all dependencies.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- node 12.x or latest
+- npm 7.x or latest
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- cd my-folder
+- git clone git@github.com:Artufelii/microverse-hello-world
+
+### Install
+
+Install this project with:
+
+- cd microverse-hello-world
+- npm install
+
+### Usage
+
+To run the linters, execute the following command:
+
+- HTML: npx hint .
+- CSS: npx stylelint "**/*.{css,scss}"
+
+### Run tests
+
+To run tests, run the following command:
+
+- HTML: npx hint .
+- CSS: npx stylelint "**/*.{css,scss}"
+
+### Deployment
+
+You can deploy this project using:
+
+- You can deploy this web app directly to github pages or any other host that you want.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@Artufelii](https://github.com/Artufelii)
+- LinkedIn: [Arturo Alvarez](https://www.linkedin.com/in/arturo-alvarez-martinez-ab9b77151/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
```

Comment:
- [ ] The `FAQ` is the only optional section as said by your previous reviewer. if you don't intend to add to it, kindly delete the whole section. Else, I strongly advice you to add a possible question people might ask about your project